src/acl_device.cpp

```diff
@@ -251,13 +263,9 @@ CL_API_ENTRY cl_int CL_API_CALL clGetDeviceInfoIntelFPGA(
     RESULT_UINT(acl_platform.max_constant_args);
     break;
 
-  // "desktop" profile says global memory must be at least 128MB
-  // "embedded" profile says global memory must be at least 1MB
   case CL_DEVICE_MAX_CONSTANT_BUFFER_SIZE: {
-    // Constant memory is global memory.
-    // However conformance_test_api min_max_constant_buffer_size
```

Comment:
Conformance test test_min_max_constant_buffer_size allocation size has been fixed by [this commit](https://github.com/KhronosGroup/OpenCL-CTS/commit/f6d6f6b3cc6f4f44ef4049b2e957276cc80615cb).